metadata-ingestion/src/datahub/sdk/lineage_client.py

```diff
@@ -36,7 +44,7 @@ def _get_fields_from_dataset_urn(self, dataset_urn: DatasetUrn) -> Set[str]:
             str(dataset_urn), SchemaMetadataClass
         )
         if schema_metadata is None:
-            return Set()
+            return set()
```

Comment:
updated since this throws typing error

```   
raise TypeError(f"Type {self._name} cannot be instantiated; "
TypeError: Type Set cannot be instantiated; use set() instead
```